### ISSUE TITLE
Reduce data downloaded by tests

### ISSUE THREAD

**Issue by dstansby** [MAVENSDC/cdflib#138]
@bryan-harter there seem to be some new tests of the xarray reader/writer that download pretty large files, making the tests run fairly slowly. Would it be possible to use the small PSP file I've included as test data instead for some of these tests?

**Comment by bryan-harter** [comment]
Yeah I might comment out a few of the larger ones.  It does take awhile to run, but on the bright side it really tests out cdflib in a bunch of places!

**Comment by dstansby** [comment]
Great! I think adding the remote data option is a good compromise - maybe I can add a separate test run that does the remote data stuff.
